src/plugins/typechecker/README.md

```diff
@@ -195,8 +212,5 @@ cabal install 'base >=4.9 && <4.12' 'containers >=0.5 && <0.6' \
 
 ## Limitations
 
-- Rather experimental
-- Type checking only happens when getting or setting
-a key in a mounted specification
+- Type checking only happens when getting or setting a key in a mounted specification
```

Comment:
Which automatically happens if cascading keys are used, isn't it? This should be clarified here.

Reply:
clarified